### DISCUSSION TITLE
Help needed: battery replacement not charging, stuck

### DISCUSSION THREAD

**Discussion by gabriele2000** [VR-25/acc#138] [Q&A] [answered]
Basically I ordered a higher capacity battery for my phone, a Sony Xperia XA1 Dual (G3112), it failed to arrive in time, i requested a refund... money arrived, but today, months later, the battery arrived.

I was excited and, most of all, surprised to say the least. I plugged it in immediately and I noticed strange things:
-The device would power on but when the boot was completed the phone immediately shut down itself, like when you press "shutdown".
-The device, while being off, would not charge at all, the battery icon would remain stuck.

The original battery has a capacity of 2300mAh while the high-capacity one has 5150mAh.
The voltage is slightly different, the original one is 3.8V and the high-capacity one is "Rated voltage 3.85V - Charging limit voltage 4.4V".

I guess that the system shuts itself due to the reading not being made at all, so I am asking you:
Is it possibile to bypass the auto-shutdown thing on android 8.0? Will acc help?

**Comment by VR-25** [comment, marked as answer]
I missed this one!
It's been a while...
Anyway, either this battery is faulty or not compatible at all.
You can try to prevent shutdown by setting capacity_sync=true (`acc -s cs=true`).
